app/policies/consent_reminder_policy.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class ConsentReminderPolicy < ApplicationPolicy
```

Comment:
I don't think we need this class as it's got no logic in it, we can then remove the corresponding `authorize :consent_reminder` calls in the controller.

Generally a `Policy` class would map to a model, in this case we don't have a `ConsentReminder` model.